src/config/configOptionMigrations.js

```diff
@@ -37,6 +37,10 @@ module.exports = {
       'platform.drive.abci.log.level',
     );
 
+    // delete an option that should not exist after the migration
+    // eslint-disable-next-line no-param-reassign
+    delete options.platform.drive.abci.log.level;
```

Comment:
There is no point to delete this option since it will be overwritten with

```js
lodashSet(options, 'platform.drive.abci.log', baseConfig.platform.drive.abci.log);
```